diff --git a/available-plugins/export/etc/jenv.d/init/export_jenv_hook.zsh b/available-plugins/export/etc/jenv.d/init/export_jenv_hook.zsh
index 52c5f58..b689546 100644
--- a/available-plugins/export/etc/jenv.d/init/export_jenv_hook.zsh
+++ b/available-plugins/export/etc/jenv.d/init/export_jenv_hook.zsh
@@ -21,4 +21,4 @@ function install_hook {
 }
 install_hook

-_jenv_export_hook
+#_jenv_export_hook
